app/assets/stylesheets/_mixins.scss

```diff
@@ -44,3 +44,17 @@
     color: inherit
   }
 }
+
+
+
+// Card styling
+@mixin dev-card {
```

Comment:
I'm getting extra thick borders on the article show page in the default styling (not with the themes). Not sure if that's intentional.

![](https://p78.f0.n0.cdn.getcloudapp.com/items/NQuv2q9g/Image+2020-02-07+at+11.21.53+AM.png?v=31004d857cbbc2e8df48e085be12731b)

Reply:
Definitely not. I'll take a look. I still have to QA other views.

Reply:
I fixed this one and couple more things on this view. Also few other "cards" stylings here and there...